pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/IngestionUtils.java

```diff
@@ -203,11 +204,16 @@ public static void uploadSegment(String tableNameWithType, BatchConfig batchConf
 
     List<String> segmentTarURIStrs = segmentTarURIs.stream().map(URI::toString).collect(Collectors.toList());
     String pushMode = batchConfig.getPushMode();
+    Map<URI, String> uriToLineageEntryIdMap = new HashMap<>();
```

Comment:
Looks like this function is only used in test. The push job runner should be wired with consistent push protocol, we won't need the change here?

Reply:
Thanks, changes removed. I also see that it's used in `PinotSinkFunction`, but that is only used for REALTIME table afaik.